src/Sarif.Converters/PREFastObjectModel/Category.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters.PREFastObjectModel
+{
+    public class Category : Dictionary<string, string>, IXmlSerializable
+    {
+        public XmlSchema GetSchema()
+        {
+            return null;
+        }
+
+        public void ReadXml(XmlReader reader)
+        {
```

Comment:
Can't deserialize to a dictionary unless you tell it how

Reply:
If it's worth a comment in the PR review, it's worth a comment in the source code.